src/main/java/com/onarandombox/MultiversePortals/commands/ListCommand.java

```diff
@@ -19,10 +20,12 @@
 
 public class ListCommand extends PortalCommand {
 
+    int itemsPerPage = 18;
```

Comment:
From what I recall Minecraft client only shows 10 lines, so maybe lines per page need to reduce.

Reply:
I, personally, would prefer to see more as 9 (10 - headline) lines. 
I would rather open the chat to see more, normally you are typing a command right after doing `/mvp list`, so the chat is open anyway.
But you're the reviewer, you have to decide.

Reply:
Let change to 9 as you mentioned with 10 as the header - the reason being mv-core `/mv list` command caps at 10 lines as well.